packages/virtual-core/src/index.ts

```diff
@@ -334,6 +337,7 @@ export class Virtualizer<
   scrollElement: TScrollElement | null = null
   targetWindow: (Window & typeof globalThis) | null = null
   isScrolling = false
+  debounceFallbackEnabled = false
```

Comment:
I think `debounceFallbackEnabled` is to generic name, what about?
```suggestion
  scrollendEventEnabled = true
```